sal_interfaces/Scheduler/Scheduler_Events.xml

```diff
@@ -82,114 +166,114 @@
     </item>
 </SALEvent>
 <SALEvent>
-    <Subsystem>MTScheduler</Subsystem>
-    <Version>3.7</Version>
+    <Subsystem>Scheduler</Subsystem>
+    <Version>3.8.13</Version>
     <Author></Author>
-    <EFDB_Topic>MTScheduler_logevent_settingsApplied</EFDB_Topic>
+    <EFDB_Topic>Scheduler_logevent_settingsApplied</EFDB_Topic>
     <Alias>settingsApplied</Alias>
     <Explanation>http://sal.lsst.org</Explanation>
     <item>
         <EFDB_Name>version</EFDB_Name>
         <Description></Description>
         <IDL_Type>string</IDL_Type>
-        <IDL_Size>256</IDL_Size>
+        <IDL_Size>1</IDL_Size>
         <Units></Units>
         <Count>1</Count>
     </item>
     <item>
-        <EFDB_Name>MTScheduler</EFDB_Name>
+        <EFDB_Name>scheduler</EFDB_Name>
         <Description></Description>
         <IDL_Type>string</IDL_Type>
-        <IDL_Size>256</IDL_Size>
+        <IDL_Size>1</IDL_Size>
         <Units></Units>
         <Count>1</Count>
     </item>
     <item>
         <EFDB_Name>cloudModel</EFDB_Name>
         <Description></Description>
         <IDL_Type>string</IDL_Type>
-        <IDL_Size>256</IDL_Size>
+        <IDL_Size>1</IDL_Size>
         <Units></Units>
         <Count>1</Count>
     </item>
     <item>
         <EFDB_Name>downtimeModel</EFDB_Name>
         <Description></Description>
         <IDL_Type>string</IDL_Type>
-        <IDL_Size>256</IDL_Size>
+        <IDL_Size>1</IDL_Size>
         <Units></Units>
         <Count>1</Count>
     </item>
     <item>
         <EFDB_Name>seeingModel</EFDB_Name>
         <Description></Description>
         <IDL_Type>string</IDL_Type>
-        <IDL_Size>256</IDL_Size>
+        <IDL_Size>1</IDL_Size>
         <Units></Units>
         <Count>1</Count>
     </item>
     <item>
         <EFDB_Name>skybrightnessModel</EFDB_Name>
         <Description></Description>
         <IDL_Type>string</IDL_Type>
-        <IDL_Size>256</IDL_Size>
+        <IDL_Size>1</IDL_Size>
         <Units></Units>
         <Count>1</Count>
     </item>
     <item>
         <EFDB_Name>observatoryModel</EFDB_Name>
         <Description></Description>
         <IDL_Type>string</IDL_Type>
-        <IDL_Size>256</IDL_Size>
+        <IDL_Size>1</IDL_Size>
         <Units></Units>
         <Count>1</Count>
     </item>
     <item>
         <EFDB_Name>observatoryLocation</EFDB_Name>
         <Description></Description>
         <IDL_Type>string</IDL_Type>
-        <IDL_Size>256</IDL_Size>
+        <IDL_Size>1</IDL_Size>
         <Units></Units>
         <Count>1</Count>
     </item>
 </SALEvent>
 <SALEvent>
-    <Subsystem>MTScheduler</Subsystem>
-    <Version>3.7</Version>
+    <Subsystem>Scheduler</Subsystem>
+    <Version>3.8.13</Version>
     <Author></Author>
-    <EFDB_Topic>MTScheduler_logevent_validSettings</EFDB_Topic>
+    <EFDB_Topic>Scheduler_logevent_validSettings</EFDB_Topic>
```

Comment:
I think this topic (`validSettings`) can be removed.

Reply:
Ok I will remove this topic.